<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandConstructFind</class>
 <widget class="QDialog" name="DialogCommandConstructFind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Поиск по имени</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Поиск по размеру</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Поиск по времени модификации</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Поиск по режиму доступа</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="nonSymbolic">
     <property name="text">
      <string>Никаких символических файлов</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="fullNames">
     <property name="text">
      <string>Полные имена файлов</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="onlyFiles">
     <property name="text">
      <string>Только файлы</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="constructButton">
     <property name="text">
      <string>construct</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameInput"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="permInput"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="timeInput"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="sizeInput"/>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="register_2">
     <property name="text">
      <string>Учитывать регистр</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
